### ISSUE TITLE
Docker image gcr.io/prysmaticlabs/prysm/slasher:v1.4.2 don't exists

### ISSUE THREAD

**Issue by Ripolin** [prysmaticlabs/prysm#9265]
Docker image ```gcr.io/prysmaticlabs/prysm/slasher:v1.4.2``` not found.

When trying to pull it I've got this message:

```log
Error response from daemon: manifest for gcr.io/prysmaticlabs/prysm/slasher:v1.4.2 not found: manifest unknown: Failed to fetch "v1.4.2" from request "/v2/prysmaticlabs/prysm/slasher/manifests/v1.4.2".
```

**Comment by rkapka** [comment]
cc @prestonvanloon

**Comment by rauljordan** [comment]
Hi @Ripolin for the last few release notes, we have communicated that slasher will be deprecated as it is no longer supported by our team. There will be an updated slasher for Prysm v2.0.0 that users can try, but the current slasher is deprecated and will no longer have docker images nor binaries released. Thank you for your understanding.

https://github.com/prysmaticlabs/prysm/releases/tag/v1.4.1

**Comment by Ripolin** [comment]
Thx for your response  @rauljordan

I understand you deprecate slasher in your next iteration but to me it's not very logical to release the binary file https://github.com/prysmaticlabs/prysm/releases/download/v1.4.2/slasher-v1.4.2-linux-amd64 for version 1.4.2 and not provide a container image like other components (validator and beacon node)

**Comment by terorie** [comment]
@rauljordan The v1.4.1 release states

> Prysm slasher binary deprecation
We have been communicating that the slasher is going to be deprecated in the v1.4 release. This effort has been pushed back while we work on various storage optimizations in parallel to Altair development. Expect the Prysm slasher binary will be deprecated and removed in the upcoming Prysm v2 release with Altair support. Follow the v2 tracking issue for more info #9166.

As v1.4.1 has not deprecated slasher, there's a `gcr.io` Docker image for slasher too.

The v1.4.2 release just states the following:

> Small fix on top of v1.4.1. Please review the v1.4.1 release notes if upgrading from v1.3.x or older.

However there's no v1.4.2 slasher release.

Which contradicts your statement

> [...] the current slasher is deprecated and will no longer have docker images nor binaries released

As of now, the v1.3.x slasher configuration (which is not deprecated according to release notes) is broken on v1.4.1 and v1.4.2:
* On v1.4.1, subcommands don't work, making the release pretty much unusable. The slasher Docker image is present though.
* On v1.4.2, the slasher Docker image is missing (the binary is present)
